examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<blockquote>import os (for example)</blockquote><p>delete unused imports! (Related to all notebooks)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>